pkg/validation/config_test.go

```diff
@@ -780,6 +781,15 @@ func TestReleaseBuildConfiguration_DependencyParts(t *testing.T) {
 			expectedTag:    "installer",
 			explicit:       true,
 		},
+		{
+			name:           "explicit, long-form for stable, overridden by cluster claim via latest release name",
```

Comment:
we want test cases with non-nil ClaimRelease that show current behavior does not change